### PR TITLE
Wrike: Extract project name and issue number

### PR DESCRIPTION
**Pull request by archnode**
Please remember the [Contributing Guidelines](https://github.com/toggl/toggl-button/blob/master/.github/CONTRIBUTING.md) :heart:

## :star2: What does this PR do?

This adds further information to the description (wrike issue number) as well as the project name (folder / project name) to the toggle button integration.

### CODE REVIEW

**Comment by archnode** [comment]
Thank you for testing and your screenshot! I extract the issue number from the url that gets set when the popup is opened. Maybe a cross-browser issue - Are you testing in Chrome?

**Comment by tcrammond** [comment]
Testing in Chrome, that's right. Parameter `'ot'` doesn't seem to exist in my issue URLs. I was using a new account and did not pick any kind of workspace template during setup.

**Comment by archnode** [comment]
I'm sorry for the long delay: The problem with the parameter stems from the different task views in wrike (major oversight, new to wrike myself).

I updated the pr with:
- Moved Issue number to back of description (As this isn't as important / canonical as in other systems)
- Make Issue number optional (This enables tracking of non-issues if presented in a task view (Projects in subfolders for example)
- Fixes extraction problem with different task views.

File: src/scripts/content/wrike.js [hunk 1]
```diff
@@ -1,18 +1,45 @@
...
+        var hashParts = hash.split("&");
+        var taskId = null;
+        for (var i = 0; i < hashParts.length; i++) {
+          var partSplit = hashParts[i].split('=');
+          if (partSplit[0] == 'ot' || partSplit[0] == 't') {
+            taskId = partSplit[1];
+          }
+        }
+        // We try to extract a title
+        // var titleElem = $('.title-field-ghost', elem);
```
Comment (tcrammond) [inline src/scripts/content/wrike.js]: Please remove this if it's not needed 😇

**Comment by archnode** [comment]
Thank you for the feedback. I made the changes you suggested - the button in the pop-up looks equally misaligned, so I added the margin generally.

**Comment by archnode** [comment]
I tried my best to update the branch!

**Pull request merged**